express-back-end/routes/notifications.js

```diff
@@ -14,16 +16,70 @@ module.exports = (db) => {
       });
   });
 
-  router.get("/user/:user_id", (req, res) => {
-    db.getNotificationByUser(req.params.user_id)
-      .then(notifications => {
-        res.json({ notifications });
+  // SUBSCRIBE USER TO PUSH NOTIFICATIONS
+  router.post('/subscribe', (req, res) => {
+    const subscription = req.body.subscription;
+    // console.log("hit the subscribe route!");
+    // console.log(req.body);
+  
+    if (req.body.userID) {
+      // console.log("attempting to add push subscription to db");
+      db.addPushSubscription(req.body.userID, subscription)
+        .then((data) => {
+          console.log(`New push subscription added to user ${data.rows[0].id}`);
+        })
+        .catch(err => {
+          console.log(err);
+        })
+    }
+  
+    // console.log(subscription);
+  
+    const payload = JSON.stringify({
+      title: 'TIMBY',
+      body: 'Welcome to TIMBY!',
+    })
+  
+    webpush.sendNotification(subscription, payload)
+      .then(result => {;
+        console.log("Notification sent from server!");
+        // console.log(result);
       })
-      .catch(err => {
-        res
-          .status(500)
-          .json({ error: err.message });
-      });
+      .catch(e => console.log(e.stack))
+  
+    res.status(200).json({'success': true})
```

Comment:
Doesn't this need to be somewhere between #44 ~ #47?
If it always sets the status to 200, it'd be nice if you can explain this part in a comment!